go.mod

```diff
@@ -34,3 +34,6 @@ require (
 	golang.org/x/sys v0.0.0-20220817070843-5a390386f1f2 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+// Downgraded to avoid bugs in following commits. See https://github.com/cosmos/cosmos-sdk/pull/14952
+replace github.com/syndtr/goleveldb => github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
```

Comment:
Can we downgrade the required version? The replace doesn't really do much here.

Reply:
Done 👌